### PR TITLE
Moved the admin award year select

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

File: app/assets/stylesheets/admin/base.scss [hunk 1]
```diff
@@ -1670,3 +1670,75 @@ table .ellipsis {
...
+
+    .caret-container {
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: auto;
+      padding-right: 10px;
+      padding-left: 10px;
+      border-left: 1px solid #2e6da4;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Color literals like `#2e6da4` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: app/assets/stylesheets/admin/base.scss [hunk 2]
```diff
@@ -1670,3 +1670,75 @@ table .ellipsis {
...
+      background: #337ab7;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Color literals like `#337ab7` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: app/assets/stylesheets/admin/base.scss [hunk 3]
```diff
@@ -1670,3 +1670,75 @@ table .ellipsis {
...
+        color: $white-true;
+      }
+    }
+  }
+
+  .btn:hover,
+  .btn:active,
+  .open > .dropdown-toggle {
+    .caret-container {
+      border-color: #204d74;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Color literals like `#204d74` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: app/assets/stylesheets/admin/base.scss [hunk 4]
```diff
@@ -1670,3 +1670,75 @@ table .ellipsis {
...
+      background-color: #286090;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Color literals like `#286090` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: app/assets/stylesheets/admin/base.scss [hunk 5]
```diff
@@ -1670,3 +1670,75 @@ table .ellipsis {
...
+    }
+  }
+
+  .btn:active .caret-container {
+    background-image: none;
+    -webkit-box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+            box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Line should be indented 4 spaces, but was indented 12 spaces

File: app/assets/stylesheets/admin/base.scss [hunk 6]
```diff
@@ -1670,3 +1670,75 @@ table .ellipsis {
...
+
+.award-year-container {
+  position: relative;
+
+  .award-year-label {
+    position: absolute;
+    top: 0;
+    left: 0;
+    margin-top: 0.35em;
+    text-transform: uppercase;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered position, top, left, margin-top, font-weight, text-transform

File: app/assets/stylesheets/admin/base.scss [hunk 7]
```diff
@@ -1670,3 +1670,75 @@ table .ellipsis {
...
+  .btn:active,
+  .open > .dropdown-toggle {
+    .caret-container {
+      border-color: #204d74;
+      background-color: #286090;
+    }
+  }
+
+  .btn:active .caret-container {
+    background-image: none;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered outline, background-image, -webkit-box-shadow, box-shadow

File: app/assets/stylesheets/admin/base.scss [hunk 8]
```diff
@@ -1670,3 +1670,75 @@ table .ellipsis {
...
+    -webkit-box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+            box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+    outline: 0;
+  }
+
+  .dropdown-menu {
+    min-width: 100px;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered left, min-width, z-index

File: app/assets/stylesheets/admin/base.scss [hunk 9]
```diff
@@ -1670,3 +1670,75 @@ table .ellipsis {
...
+      border-color: $bootstrap-blue-line-dark;
+      background-color: $bootstrap-blue-dark;
+    }
+  }
+
+  .btn:active .caret-container {
+    outline: 0;
+    background-image: none;
+    -webkit-box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+            box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Line should be indented 4 spaces, but was indented 12 spaces

Comment (hammerfunk) [inline app/assets/stylesheets/admin/base.scss]: Ignore

**Comment by RuslanHamidullin** [comment]
Looks Good :+1:

**Pull request merged**